src/transformers/models/ibert/configuration_ibert.py

```diff
@@ -123,3 +127,22 @@ def __init__(
         self.position_embedding_type = position_embedding_type
         self.quant_mode = quant_mode
         self.force_dequant = force_dequant
+
+
+class IBertOnnxConfig(OnnxConfigWithPast):
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("input_ids", {0: "batch", 1: "sequence"}),
+                ("attention_mask", {0: "batch", 1: "sequence"}),
+            ]
+        )
+
+    @property
+    def outputs(self) -> Mapping[str, Mapping[int, str]]:
```

Comment:
You do not need to override the `outputs` property as it is already taken care of by `OnnxConfig` for each of the tasks supported by IBERT (the outputs will depend on the task at hand, so the base class `outputs` property takes care of returning the proper outputs for a given task).